base/logging.jl

```diff
@@ -97,16 +129,13 @@ The log level provides a key against which potential log records may be
 filtered, before any other work is done to construct the log record data
 structure itself.
 """
-struct LogLevel
+struct LogLevel <: AbstractLogLevel
     level::Int32
```

Comment:
Can we just change this to `Int`?
Or make `severity` return an `Int32`
I feel like the overhead of converting it each time is not worth it.